api/agent/version.go

```diff
@@ -26,6 +27,10 @@ func GetAgentVersionAndPlatform(url string, tlsConfig *tls.Config) (portainer.Ag
 		}
 	}
 
+	if !strings.Contains(url, "://") {
```

Comment:
Should these changes be in this ticket?

Reply:
Needed it, or else we couldn't test as EE had this in it by default and for some reason was missing from CE.